tests/ha/test_ha_node_health_gui.py

```diff
@@ -79,7 +79,7 @@ def setup_class(cls):
 
         # required for Robot_GUI
         cls.ha_gui_obj = HAGUILibs()
-
+        cls.nw_data = None
```

Comment:
Are not all minor/major warning fixed  for this file ?
I can see some long line warnings are still there in pylint check.

Reply:
Where are you seeing those?